### PR TITLE
Testing User entity

### PR DESCRIPTION
**Pull request by Divin17**
Testing user entity consists of:
  * User creation
  * Get existing user by id

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +0,0 @@
-DATABASE_URL="postgres://user:password@localhost:5432/DB_DEVELOPMENT_DOCKER"
```
Comment (danieltonel) [inline .env.example:1]: this would remove the env.example file from the main repo. can you undo this change?

will update the README, probably there was some confusion with the renaming

File: .env.test.example [hunk 2]
```diff
@@ -1 +0,0 @@
-DATABASE_URL="postgres://user:password@localhost:5432/DB_DEVELOPMENT_DOCKER"
```
Comment (danieltonel) [inline .env.test.example:1]: this would remove the env.example file from the main repo. can you undo this change?

will update the README, probably there was some confusion with the renaming

File: .husky/_/.gitignore [hunk 3]
```diff
@@ -1 +1 @@
-_
\ No newline at end of file
+*
```
Comment (danieltonel) [inline .husky/_/.gitignore:1]: what's the reason for this change?

File: package-lock.json [hunk 4]
```diff
@@ -1,28 +1,8182 @@
 {
```
Comment (danieltonel) [inline package-lock.json:1]: can you please always use `npm ci` instead of `npm i`? this way you won't override the lockfile and you'll keep the same versions

File: yarn.lock [hunk 5]
```diff
@@ -3,21 +3,21 @@
 
 
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.12.13", "@babel/code-frame@^7.16.7":
```
Comment (danieltonel) [inline yarn.lock:5]: please use npm and not yarn for this repo. this way we can use the correct lock file

**Review by danieltonel — CHANGES_REQUESTED** [review]
looks good to me, added some comments ✌️

**Comment by Divin17** [comment]
I tried to correct all files mentioned in the comments, I hope it fits what is required!

**Comment by danieltonel** [comment]
this PR should only cover tests. there should actually only be one file part of the "Files changed". however when merging this PR we would override other stuff that shouldn't be changed

that should be the only file that's changed in the PR:
- tests/business-logic/User.test.ts

can you look into that and remove the other files?
